app/views/foods/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= form_with(model: food, url: foods_url, scope: :foods, class: 'text-white' ) do |f| %>
+  <div class="modal-header">
+    <h5 class="modal-title" id="exampleModalLabel">Create food</h5>
+  </div>
+  <div class="modal-body">
+    <% if food.errors.any? %>
+      <div id="error_explanation">
+        <h2><%= pluralize(food.errors.count, 'error') %> prohibited this post from being saved:</h2>
+
+        <ul>
+          <% food.erros.full_messages.each do |message| %>
+            <li><%= message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <div class="form-group">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+    </div>
+
+    <div class="form-group">
+      <%= f.label :measurement_unit %><br />
+      <%= f.text_field :measurement_unit , autofocus: true, autocomplete: "measurement unit " %>
+    </div>
+
+    <div class="form-group">
+      <%= f.label :price %><br />
+      <%= f.text_field :price, autofocus: true, autocomplete: "price" %>
```

Comment:
I think we should change this ```text_field``` to be of another type 🤔. I consider ```number_field``` could be more suitable. That way the user has no way to introduce an improper data type and we could also handle decimals. Here's a picture of the result when I tried to add 450.52 

![image](https://user-images.githubusercontent.com/74314917/169187120-e8c540af-bb58-4313-aca9-e4c1f86413f0.png)